Vindicta.Altis/Templates/Loadouts/Arma3_LDF/SF/A3_LDFs_TL.sqf

```diff
@@ -0,0 +1,48 @@
+removeAllWeapons this;
+removeAllItems this;
+removeAllAssignedItems this;
+removeUniform this;
+removeVest this;
+removeBackpack this;
+removeHeadgear this;
+removeGoggles this;
+
+_RandomHeadgear = selectRandom ["H_HelmetSpecB_blk"];
+this addHeadgear _RandomHeadgear;
+_RandomGoggles = selectRandom ["G_Balaclava_TI_tna_F", "G_Balaclava_TI_G_tna_F"];
+this addGoggles _RandomGoggles;
+this forceAddUniform "U_B_CTRG_Soldier_F";
+this addVest "V_CarrierRigKBT_01_heavy_EAF_F";
+this addBackpack "B_AssaultPack_eaf_F";
+
+this addWeapon "arifle_MX_GL_Black_F";
+this addPrimaryWeaponItem "muzzle_snds_65_TI_blk_F";
+this addPrimaryWeaponItem "acc_pointer_IR";
+this addPrimaryWeaponItem "optic_Hamr";
+this addPrimaryWeaponItem "30Rnd_65x39_caseless_black_mag";
+this addWeapon "hgun_Pistol_heavy_01_green_F";
+this addHandgunItem "muzzle_snds_acp";
+this addHandgunItem "acc_flashlight_pistol";
+this addHandgunItem "optic_MRD_black";
+this addHandgunItem "11Rnd_45ACP_Mag";
+
+
+for "_i" from 1 to 2 do {this addItemToUniform "FirstAidKit";};
+for "_i" from 3 to 5 do {this addItemToVest "30Rnd_65x39_caseless_black_mag";};
```

Comment:
This is weird, why from 3 to 5? that is the same as from 0 to 2? It doesn't randomly select 3 to 5 of them. IF you want random amount between 3 and 5 do this:
```suggestion
for "_i" from 0 to (3 + floor random 2) do {this addItemToVest "30Rnd_65x39_caseless_black_mag";};
```

Reply:
Yep I noticed now thanks for letting me know, already fixed it.